tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "experimentalDecorators": true
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    ".vscode-test"
+  ],
+  "files": [
```

Comment:
I was looking into this a bit earlier, doesn't the `files` config define what files `tsc` will compile? (ref: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html). 

> If no "files" property is present in a tsconfig.json, the compiler defaults to including all TypeScript (*.ts or *.tsx) files in the containing directory and subdirectories. When a "files" property is present, only the specified files are included.

I think doing this just masks the problem?